code/modules/cargo/exports/large_objects.dm

```diff
@@ -171,13 +171,13 @@
 	var/worth = 10
 	var/gases = C.air_contents.gases
 
-	worth += gases[/datum/gas/bz]*4
+	worth += gases[/datum/gas/bz]*2
 	worth += gases[/datum/gas/stimulum]*25
-	worth += gases[/datum/gas/hypernoblium]*1000
-	worth += gases[/datum/gas/miasma]*4
-	worth += gases[/datum/gas/tritium]*7
-	worth += gases[/datum/gas/pluoxium]*6
-	worth += gases[/datum/gas/nitryl]*30
+	worth += gases[/datum/gas/hypernoblium]*50
```

Comment:
hypernob is hard to make because it has that funky fusion-tier temperature/energy requirement or whatever, dabbing on it like this feels unnecessary

Reply:
if you make this you make tons of it. Meaning if you use 2n:1t:1bz you get 2 tanks out of it for over around 100k if not 250k. If you know how to do it, or know the waltz then you can p-easy make them as an atmos tech for cargo to gain well over 4 million by the one hour mark.
@Putnam3145 - Can you confirm/Enlighten me if im wrong on this?

Reply:
Furthermore, if you see above with Stim, being worth less the Nitryl. Meaning bad mixes or misstakes when making Nitryl only will be more costly depending on the goal in mind.
Just do to poor rates of sale dosnt mean it should be buffed or not reblanaced well others in its department are. - See Miasma, Toxins and Heaters
Its best to nip this one well its still a few-have-known. 233 bomb that gave 20k points was a issue that never needed to happen, so the can of 230 moles of hyper selling dosnt need that same issue as well.

Reply:
okay well this "hypernob waltz" you're talking about is some real funky stuff that only a few people know (and that small group is probably a circlejerk of atmos mains who refuse to teach anyone about assmos reactions)
and nitryl (burns lungs, slight stimulant effect(?)) being worse selling than stimulum proper (toxic, but a better stimulant effect) seems like a poor choice
why would central go "haha i really like this lung burning gas that is only slightly more useful than this funky stimulum gas that only /slightly/ kills our troops"